Makefile

```diff
@@ -3,6 +3,11 @@
 
 test: test-heroku-16
 
+check:
+	@shellcheck bin/compile bin/detect bin/release bin/test-compile bin/utils bin/warnings
```

Comment:
One option would be to use git-grep to search for relevant shebangs to avoid having to keep this list up to date, like so:
https://github.com/mozilla-platform-ops/devservices-aws/blob/1ea2f0b28d5ef23885b451873129e084d287be2b/runtests.sh#L17